albums/Quite Good Rocket Launcher Music/2. Back In Biznitch.txt

```diff
@@ -35,13 +35,13 @@ don't fuck with us or we'll fucking murk you
 
 Mos Prob gets to show off just bezerk
 you underrated us and now here's your just deserts
-[??] (we'll|will) fucking hurt you in your inner ear
+who will fucking hurt you in your inner ear
 God would save you right now, but he isn't here
 
 My willy [??] for bitches, it never plays right
 on the upside, at least their vaginas stay tight
 I'm no Shakespeare, but I'm a fucking playwright
-equilibrium, conflict, yeah, I'll die of stage fright
+equilibrium, conflict, y'all die of stage fright
```

Comment:
the rest of this verse is all self-deprecation, so it seems more likely that this line is an admission of a fault than an accusation